tasks/section4.yml

```diff
@@ -503,7 +503,7 @@
     path: "{{ item }}"
     regexp: '^(daily|weekly|monthly|yearly)$'
     line: "{{ rhel7cis_logrotate }}"
-		create: yes
+    create: yes
```

Comment:
I don't think we should do a `create: yes` here.

Reply:
The problem with `with_fileglob` is that it is actually doing the searching locally on your control machine (I just found this out right now as well)...not remotely. Its actually a lookup plugin and all lookup plugins operate locally. You can use `command: find` or you can use the `find` module, or you can use the `stat` module. But you'll have to have a task that looks for the files, registers the output, and then do a `with_items` on this `lineinfile` task.

Reply:
Good to know! :)

I'll update